src/components/field-zone/dto/field-zone.dto.ts

```diff
@@ -26,7 +25,7 @@ export class FieldZone extends Resource {
   @DbUnique()
   readonly name: SecuredString;
 
-  readonly director: Secured<ID>;
+  readonly director: Secured<StaticRef<'User'>>;
```

Comment:
This is required to allow EdgeDB to return this shape.

This shape is a subset of a User (`{ id: ID }`), whereas previously, a `User`
could not become an `ID` when selecting.
```edgeql
select FieldZone {
  director := .director.id # error
}
```
Luckily this seems to be a pretty easy change to make, and Neo4j can easily accommodate.

Reply:
Idk if `StaticRef` is the best name. It makes sense when put next to `DynamicRef`.
But the latter is rarely used, where as the former will be used a lot.
I thought about just calling it `Ref`. It's pretty generic though, and other libraries have their own "ref" concepts which could lead to ambiguity.
Perhaps another word would better describe the type & identifier combo?

Reply:
Ok I renamed this to `LinkTo`. I think that's probably the best name we came up with on zoom today. My gut says we should go this route to have the field have the same name rather than trying to follow a convention with an `Id` suffix.